ketch-ios/ConsentConfig.swift

```diff
@@ -51,20 +51,18 @@ struct ConsentConfig {
 
     private var fileUrl: URL? {
         let url = Bundle.main.url(forResource: htmlFileName, withExtension: "html")!
-
         var urlComponents = URLComponents(string: url.absoluteString)
         urlComponents?.queryItems = queryItems
 
         return urlComponents?.url
     }
 
     private var queryItems: [URLQueryItem] {
-        
-        return [
+        [
             URLQueryItem(name: "propertyName", value: propertyName),
             URLQueryItem(name: "orgCode", value: orgCode),
             URLQueryItem(name: "idfa", value: advertisingIdentifier.uuidString),
-            URLQueryItem(name: "swb_region", value: "US-CA")
+            URLQueryItem(name: "swb_region", value: NSLocale.current.regionCode) // "US-CA" ???
```

Comment:
I think this could be auto detected on the bac-end, based on public IP address of the device
the `swb_region` is used mostly by developers to force a certain region fo the debugging purposes 

I'd say we skip this parameter and rely on autodetection

Reply:
right. removed